app/src/main/java/com/codesoom/assignment/TaskHttpHandler.java

```diff
@@ -0,0 +1,81 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TaskHttpHandler implements HttpHandler {
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private final OutputStream outputStream = new ByteArrayOutputStream();
+    private List<Task> tasks = new ArrayList<>();
+
+    @Override
+    public void handle(HttpExchange httpExchange) throws IOException {
+        String method = httpExchange.getRequestMethod();
+        URI uri = httpExchange.getRequestURI();
+        String path = uri.getPath();
+
+        InputStream inputStream = httpExchange.getRequestBody();
+        String body = new BufferedReader(new InputStreamReader(inputStream)).lines().collect(Collectors.joining("\n"));
```

Comment:
한 줄이 너무 길어요. 코드를 리뷰하는 동료를 위해 적절히 개행해 주세요. 보통은 이렇게 `.`을 기준으로 개행해 줍니다.
```suggestion
        String body = new BufferedReader(new InputStreamReader(inputStream))
            .lines()
            .collect(Collectors.joining("\n"));
```

Reply:
넵! . 을 기준으로 개행하니까 제가보기에도 좀더 깔끔함이 느껴집니다